app/classes/Update.php

```diff
@@ -601,5 +606,23 @@ private function schema23()
             throw new Exception('Cannot change type of column "body" in table "items_revisions"!');
         }
     }
-
+  
```

Comment:
install an addon for your file editor that shows the trailing whitespace. It helps a lot. Also one for doing PSR parsing on each save. Here there are two whitespaces.